errbot/backends/telegram_messenger.py

```diff
@@ -256,6 +258,8 @@ def _handle_message(self, message):
             log.warning("Unhandled message type (not a text message) ignored")
             return
 
+        if re.match('\/.[^ ]*@{}'.format(self.botname), message.text):
```

Comment:
`/` is not a special character in PCRE, there should be no need for escaping. Regexes are typically also entered using a raw litereal to avoid newline escapes and such: `re.match(r"/.[^ ]*@{}".format(self.botname), message.text)`